draft-ietf-tls-dtls13.md

```diff
@@ -300,6 +307,44 @@ length:
 encrypted_record:
 : Identical to the encrypted_record field in a TLS 1.3 record.
 
+{:br}
+
+The short DTLS header format is:
+
+~~~~
+    struct {
+      uint16 short_epoch_and_sequence;  // 001ESSSS SSSSSSSS
+      opaque encrypted_record[remainder_of_datagram];
+    } DTLSShortCiphertext;
+~~~~
+
+The short_epoch_and_sequence document contains the epoch and sequence
+packed into a 16 bit integer as follows:
+
+- The first three bits are set to 001 in order to allow multiplexing
+  between DTLS and VoIP protocols (STUN, RTP/RTCP, etc.) {{?RFC7983}}
+  and distinguish the short from long header formats.
+
+- The fourth bit is the low order bit of the epoch value.
+
+- The remaining bits contain the low order 12 bits of the sequence
+  number.
+
+For deprotecion purposes, the nonce is computed using the full
```

Comment:
deprotection doesn't appear to be a word, maybe "when removing packet protection, the nonce is computed..."